about us/about_us.html

```diff
@@ -5,9 +5,47 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="about_us.css" />
+    <script defer src="about_us.js"></script>
     <title>About Us</title>
   </head>
   <body>
     <h1>Get familiar with our program</h1>
+    <h2>Intro</h2>
+    <p>
+      “2nd Chance” program is looking to help people who have the talent/ability
+      and the fortitude to become engineers, but never had the opportunity to do
+      so. We start with minimal support. As you prove to be dedicated and
+      capable, we will increase the amount of support. Support includes
+      financing the course you will take, providing equipment, and for the
+      senior candidates even paying living expenses so that you can focus on
+      studies full time. Most importantly, support includes time with a mentor,
+      who is on the same track, but ahead of you by at least 2 years. To be
+      clear, we don&apos;t magically do this instead of you. You put in the work
+      and you earn the opportunity to be an engineer. We are just here to make
+      the transition easier and faster.
+    </p>
+    <h2>What we look for in a candidate</h2>
+    <div id="candidate-traits"></div>
+    <p class="p-underline">Below are the stages of the program.</p>
```

Comment:
instead of using your styling method as a class name, e.g. p-underline, use the function, e.g. program-stages-guide. This way if you decide to change this paragraph from being underlined to bold for example, you won't have to rename the class to p-bold.
Also, should this one liner be a paragraph or a header?